pkg/dns/vips/virtual_outbound_view.go

```diff
@@ -46,7 +46,11 @@ func (vo *VirtualOutboundMeshView) Add(entry HostnameEntry, outbound OutboundEnt
 			if existingOutbound.String() == outbound.String() {
 				return nil
 			}
-			return fmt.Errorf("can't add %s:%d from %s because it's already used by entity defined in:'%s'", entry.Name, outbound.Port, outbound.Origin, existingOutbound.Origin)
+			if entry.Type == Host {
+				return fmt.Errorf("autogenerated DNS entry %s:%d conflicts with existing entry, this happens when there are two external services with the same 'networking.address', to disable automatic DNS generation in external services set 'skipVIPGeneration' to 'true' under 'networking'", entry.Name, outbound.Port)
+			} else {
+				return fmt.Errorf("can't add %s:%d from %s because it's already used by entity defined in:'%s'", entry.Name, outbound.Port, outbound.Origin, existingOutbound.Origin)
```

Comment:
Do we want to update this as well?

Reply:
This is only for `FQDN` and `Service` which in the case of service will be a bug, in the case of FQDN the Origin should show the right info so no I don't think you need more here.